@kiva/kv-components/vue/KvLendCta.vue

```diff
@@ -79,11 +123,12 @@
 				</div>
 
 				<!-- Lend button -->
-				<div :class="{ 'lendButtonWrapper': hideShowLendDropdown }">
+				<div :class="{ 'lendButtonWrapper': hideShowLendDropdown && !showPresetAmounts}">
```

Comment:
Also to make sure these use cases are still covered, could you add preset stories for: loading, adding, and basketed? Note that the ticket calls out that it should say, for example, "Continue to basket".